hw/xbox/xbox.c

```diff
@@ -204,7 +204,8 @@ static void xbox_flash_init(MemoryRegion *rom_memory)
          map_loc >= 0xff000000;
          map_loc -= bios_size) {
         map_bios = g_malloc(sizeof(*map_bios));
-        memory_region_init_alias(map_bios, NULL, "pci-bios", bios, 0, bios_size);
+        memory_region_init_alias(map_bios, NULL, "pci-bios",
+                                 bios, 0, bios_size);
```

Comment:
yuk.. I liked the old one better - that's also why I hate the 80 column limit (when almost nobody uses 80 columns anymore)

*Just wanted to rant, change is acceptable - it's a good implementation of a stupid rule.*